modules/apps/portal-workflow/portal-workflow-test/src/testFunctional/tests/WorkflowUser.testcase

```diff
@@ -25,6 +25,51 @@ definition {
 		}
 	}
 
+	@description = "This is a test for LRQA-69712. Verify that the user can approve a new account by Notifications. "
+	@ignore = "Test Stub"
+	@priority = "3"
+	test CanApproveByDetails {
```

Comment:
Hi kristin! Could you, please, change the test name to "CanApproveByNotifications"? Thank you! :)

Reply:
Changed the test name!